### PR TITLE
@types/svgo: fix type issues with PrefixIdsPlugin

### PR DESCRIPTION
**Pull request by Scribblerockerz**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [svgo/plugins/prefixIds.js:78](https://github.com/svg/svgo/blob/a9834efa1603198ae606765292215462ae193d0d/plugins/prefixIds.js#L78)
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header. (no version change)

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## svgo ([<kbd>unpkg</kbd>](https://unpkg.com/browse/svgo@latest/))
was missing the following properties:
1. createContentItem
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against a52cdf328ac292cfe8f906c1133eca0af1b37c3e
</p>

File: types/svgo/index.d.ts [hunk 1]
```diff
@@ -314,6 +314,7 @@ export type PluginsPlugin = DefaultPlugin<'plugins'>;
 export type PrefixIdsPlugin = DefaultPlugin<
     'prefixIds',
     {
+        prefix?: boolean | string | ((node: any, info: any) => string),
```
Comment (peterblazejewicz) [inline types/svgo/index.d.ts:317]:
```suggestion
        prefix?: boolean | string | ((node: any, info: any) => string) | undefined;
```
I believe this should work at runtime, to keep with convention here and to get TS4.4 strict optional properties support when used

Comment (Scribblerockerz) [reply types/svgo/index.d.ts]: Added your suggestion, thanks!

**Comment by petrzjunior** [comment]
I see in the upstream JSDoc:
```ts
prefix?: boolean | string | ((node: XastElement, info: PluginInfo) => string),
```
Could we add the types `XastElement` and `PluginInfo` as well to replace `any`?

**Comment by Scribblerockerz** [comment]
@petrzjunior added `XastElement` and `PluginInfo` from [svgo/lib/types.ts](https://github.com/svg/svgo/blob/a9834efa1603198ae606765292215462ae193d0d/lib/types.ts#L30)

Is it right or did I miss there something?

**Review by petrzjunior — APPROVED** [review]
Look good to me now

**Comment by Scribblerockerz** [comment]
Ready to merge

**Pull request merged**